README.md

```diff
@@ -35,6 +35,9 @@ optional dependencies:
 2) Build SCAP Workbench:
 ```console
 $ mkdir build; cd build
+$ # This export line is only required if openscap was compiled locally;
+$ # add to PKG_CONFIG_PATH the directory containing libopenscap.pc
+$ export PKG_CONFIG_PATH=$PKG_CONFIG_PATH:/PATH/TO/DIR/WITH/.pcFILE/
```

Comment:
Please consider these adjustments:
* subsitute "containing libopenscap.pc" with something like "containing _installed_ libopenscap.pc _(typically located in a directory named 'pkgconfig')_", so users can search for it and find the right one quickly, and
* double quote the assignment (like `X="$X:some other stuff"`) so it works even for directories containing spaces.

I have no further ideas how to improve your contribution before merging it.